### DISCUSSION TITLE
How to get Email notifications when github actions fail?

### DISCUSSION THREAD

**Discussion by taojintianxia** [apache/skywalking#9143] [Q&A] [answered]
hi dear skywalking team

to ensure the github action works well, whenever we commit a github action PR , we will keep on tracking the action results for days,  that's no a smart way , I believe.

as we can see , there are lot of github actions in skywalking project, then how does the skywalking team to get the email when github action failed ? is there any group to subscribe or is there anyway to add some receiver to get the result of github action ?

thank you

**Comment by wu-sheng** [comment, marked as answer]
Why do you want to watch the GitHub action?
The committer team could get the notifications. But usually people don't want this kind of mails, they are too many.

**Reply by wu-sheng** [reply]
If you are the author of a pull request, you would be notified too.

**Comment by Superskyyy** [comment]
I think whenever an action fails it will only notify the original creator/ enabler (schedule/PR), it's not meant for everyone to be notified.

I can tell such a feature can be implemented by yourself very easily by polling [GitHub CLI](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/viewing-workflow-run-history), then distribute your alarms using a smtp service.

`Notifications for scheduled workflows are sent to the user who initially created the workflow. If a different user updates the cron syntax in the workflow file, subsequent notifications will be sent to that user instead. `
